lib/interfaces/express.js

```diff
@@ -46,10 +47,10 @@ function expressErrorHandler(client, config, err, req, res, next) {
   var ctxService = '';
   var ctxVersion = '';
 
-  if (isPlainObject(config) && isPlainObject(config.serviceContext)) {
+  if (isObject(config) && isFunction(config.getServiceContext)) {
```

Comment:
I think more generally, typechecks of this form can be replaced with unit tests. Validating input crossing the boundary from user code into our module makes sense but I think typechecking our own internal representations is better validated by unit tests to allow us to catch issues during development as opposed to at runtime.